package.json

```diff
@@ -27,14 +28,16 @@
     "@bcrs-shared-components/date-picker": "1.2.15",
     "@bcrs-shared-components/document-delivery": "1.2.0",
     "@bcrs-shared-components/effective-date-time": "1.1.15",
-    "@bcrs-shared-components/enums": "1.0.38",
+    "@bcrs-shared-components/fee-summary": "2.0.0",
+    "@bcrs-shared-components/enums": "1.0.46",
     "@bcrs-shared-components/genesys-web-message": "1.0.0",
     "@bcrs-shared-components/help-business-number": "1.1.1",
-    "@bcrs-shared-components/interfaces": "1.0.60",
+    "@bcrs-shared-components/interfaces": "1.0.70",
     "@bcrs-shared-components/limited-restoration-panel": "1.0.5",
-    "@bcrs-shared-components/mixins": "1.1.10",
+    "@bcrs-shared-components/mixins": "1.1.20",
```

Comment:
I've also updated a bunch of the useful packages as above like the enums, interfaces, and mixins.